### PR TITLE
ADDR-62 Use Daemon to authenticate task instead of scheduler.username and scheduler.password

### PR DESCRIPTION
**Pull request by dulvinw**
Initial commit for changes to see if the path I'm following is correct. Need to add testcases.

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you take a second look at this? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: hier.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+United States|Massachusetts|Suffolk County|Boston
+United States|Massachusetts|Plymouth County|Scituate
```
Comment (dkayiwa) [inline hier.txt]: Do you need the changes in this file?

Comment (dulvinw) [reply hier.txt]: I do not think so. I'll go through the document and re create a pull request and let you know. Thanks for the help

**Comment by dkayiwa** [comment]
@dulvinw are you still working on this?

**Pull request closed without merging**